timemachine/parallel/client.py

```diff
@@ -222,6 +226,10 @@ def verify(self):
         """
         gpus = get_gpu_count()
         assert self.max_workers <= gpus, f"More workers '{self.max_workers}' requested than GPUs '{gpus}'"
+        assert (
+            len(self._gpu_list) >= self.max_workers
+        ), "Fewer available GPUs than max workers expects, check CUDA_VISIBLE_DEVICES"
+        assert len(self._gpu_list) <= gpus, "More GPUs available than the machine has, check CUDA_VISIBLE_DEVICES"
```

Comment:
I'm a bit confused by this; should it instead be
```suggestion
        assert len(self._gpu_list) <= gpus, "More GPUs requested than the machine has, check CUDA_VISIBLE_DEVICES"
```
?